onboarding/src/main/java/eu/arrowhead/core/onboarding/OnboardingService.java

```diff
@@ -109,27 +121,63 @@ public CertificateSigningResponse signCertificate(final JcaPKCS10CertificationRe
 
     public ServiceEndpoint[] getEndpoints() throws URISyntaxException {
 
-        final List<ServiceEndpoint> endpoints = new ArrayList<>();
+        log.info("getting endpoints...");
```

Comment:
The message may be more specific, where it is originated from.

Reply:
Corrected in [3e2b6e3](https://github.com/arrowhead-f/core-java/pull/55/commits/3e2b6e3c42b5c5fbb6fd968e896bdce90fbe9e69)